src/sound_openal.cpp

```diff
@@ -511,8 +511,7 @@ class OpenALSoundManager: public ISoundManager
 			{
 				ALint state;
 				alGetSourcei(sound->source_id, AL_SOURCE_STATE, &state);
-				if(state != AL_PLAYING && state != AL_PAUSED)
-				{
+				if(state != AL_PLAYING && state != AL_PAUSED) {
```

Comment:
Oh sorry about that @nerzhul I'm really new here.
I'm reading the code guidelines now (What a shame).
I hope there's no more heresies left.